src/data-publication/constants/one-year-datasets.js

```diff
@@ -1,4 +1,33 @@
 export const ONE_YEAR_DATASET = {
+  2021: {
+    freezeDate: 'May 1, 2022',
```

Comment:
I think this was also corrected somewhere else
```suggestion
    freezeDate: 'May 1, 2023',
```

Reply:
@ojbravo Can you confirm this?

Reply:
@meissadia Good catch! This was the same update Abhi cited as well.